deploy/00-deploy-mocks.ts

```diff
@@ -1,34 +1,35 @@
-import { HardhatRuntimeEnvironment } from "hardhat/types"
-import { DeployFunction } from "hardhat-deploy/types"
+import { HardhatRuntimeEnvironment } from 'hardhat/types'
+import { DeployFunction } from 'hardhat-deploy/types'
+import { network } from 'hardhat'
 
-const DECIMALS = "18"
-const INITIAL_PRICE = "2000000000000000000000" // 2000
-const deployMocks: DeployFunction = async function (
-  hre: HardhatRuntimeEnvironment
-) {
-  const { deployments, getNamedAccounts, network } = hre
-  const { deploy, log } = deployments
-  const { deployer } = await getNamedAccounts()
-  const chainId = network.config.chainId
-  // If we are on a local development network, we need to deploy mocks!
-  if (chainId == 31337) {
-    log("Local network detected! Deploying mocks...")
-    await deploy("MockV3Aggregator", {
-      contract: "MockV3Aggregator",
-      from: deployer,
-      log: true,
-      args: [DECIMALS, INITIAL_PRICE],
-    })
-    log("Mocks Deployed!")
-    log("----------------------------------")
-    log(
-      "You are deploying to a local network, you'll need a local network running to interact"
-    )
-    log(
-      "Please run `yarn hardhat console` to interact with the deployed smart contracts!"
-    )
-    log("----------------------------------")
-  }
+import {
+    developmentChains,
+    DECIMALS,
+    INITIAL_ANSWER
+} from '../helper-hardhat-config'
+
+const deployFunction: DeployFunction = async function ({
+    getNamedAccounts,
+    deployments
+}: HardhatRuntimeEnvironment) {
+    const { deploy, log } = deployments
+    const { deployer } = await getNamedAccounts()
+
+    if (developmentChains.includes(network.name)) {
+        log('Local network detected! Deploying mocks...')
+
+        await deploy('MockV3Aggregator', {
+            contract: 'MockV3Aggregator',
+            from: deployer,
+            args: [DECIMALS, INITIAL_ANSWER],
+            log: true
+        })
+
+        log('Mocks deployed!')
+        log('---------------------------')
+    }
 }
-export default deployMocks
-deployMocks.tags = ["all", "mocks"]
+
+export default deployFunction
```

Comment:
I don't like how both deploys have the same export, I don't think this makes sense. They should be different so people can know which deploy they are working with.